### DISCUSSION TITLE
Support for Elastic UI

### DISCUSSION THREAD

**Discussion by eduardocp** [refinedev/refine#3471] [Integration request]
### Is your feature request related to a problem? Please describe.

[Elastic UI](https://elastic.github.io/eui/#/) is the front-end framework used by all products of elastic group. I think this can be a huge addition to refine's portifolio.

### Describe the thing to improve

I would like to see as many components as possible being used. They have a lot of components that are very well developed and of great value for any current interface.

**Comment by omeraplak** [comment]
Hey @eduardocp 👋,
Thank you for contacting us! Lets see what we can do 🎁

**Comment by eduardocp** [comment]
Hi, how are you guys?

Any news on this @omeraplak ?

**Comment by omeraplak** [comment]
Hey @eduardocp ,
We haven't planned this yet. It looks like we won't be getting time for this anytime soon. Does anyone want to do it on Community? ❤️
